Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -1,43 +1,64 @@
 /-
-Copyright (c) 2025 Mitchell Horner. All rights reserved.
+Copyright (c) 2023 Yaël Dillies, Mitchell Horner. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Mitchell Horner
+Authors: Yaël Dillies, Mitchell Horner
 -/
 import Mathlib.Combinatorics.SimpleGraph.Subgraph
 
 /-!
-# Copies of Simple Graphs
+# Containment of graphs
 
 This file introduces the concept of one simple graph containing a copy of another.
 
-## Main definitions
+For two simple graph `G` and `H`, a *copy* of `G` in `H` is a (not necessarily induced) subgraph of
+`H` isomorphic to `G`.
 
+If there exists a copy of `G` in `H`, we say that `H` *contains* `G`. This is equivalent to saying
+that there is an injective graph homomorphism `G → H` them (this is **not** the same as a graph
+embedding, as we do not require the subgraph to be induced).
+
+If there exists an induced copy of `G` in `H`, we say that `H` *inducingly contains* `G`. This is
+equivalent to saying that there is a graph embedding `G ↪ H`.
+
+## Main declarations
+
+Containment:
 * `SimpleGraph.Copy A B` is the type of copies of `A` in `B`, implemented as the subtype of
   *injective* homomorphisms.
-
 * `SimpleGraph.IsContained A B`, `A ⊑ B` is the relation that `B` contains a copy of `A`, that
   is, the type of copies of `A` in `B` is nonempty. This is equivalent to the existence of an
   isomorphism from `A` to a subgraph of `B`.
-
   This is similar to `SimpleGraph.IsSubgraph` except that the simple graphs here need not have the
   same underlying vertex type.
-
 * `SimpleGraph.Free` is the predicate that `B` is `A`-free, that is, `B` does not contain a copy of
   `A`. This is the negation of `SimpleGraph.IsContained` implemented for convenience.
 
+Induced containment:
+* Induced copies of `G` inside `H` are already defined as `G ↪g H`.
+* `SimpleGraph.IsIndContained G H` : `G` is contained as an induced subgraph in `H`.
+
 ## Notation
 
 The following notation is declared in locale `SimpleGraph`:
-
 * `G ⊑ H` for `SimpleGraph.IsContained G H`.
+* `G ⊴ H` for `SimpleGraph.IsIndContained G H`.
```

Comment:
TODO: 
`(top : SimpleGraph V) ⊴ H` iff `(top : SimpleGraph V) ⊑ H` (iff something about cliques too)

Reply:
How do you want the clique lemma to be stated? I'm not sure myself

Reply:
Me neither, you can just say "relate this to cliques" or something, if you like. And now that I think about it, induced containment of bot should relate to indep sets too